### PR TITLE
View resume functionality added

### PR DESCRIPTION
**Pull request by ShivangiRai1310**
Fixes: #13 

Have done the required changes in 
action.yml, 
generate_index.py, 
level1.py, 
level2.py.

### CODE REVIEW

File: generate_index.py [hunk 1]
```diff
@@ -91,11 +92,11 @@
```
Comment (kaustubhgupta) [inline generate_index.py:92]: You have not handled the case where the user doesn't provide any resume URL. In that case, you should not display any "view resume" text on the webpage. See the logics for blogs and hackathon to get an idea about how to handle it

File: themes/level_1/level1.py [hunk 2]
```diff
@@ -16,7 +16,8 @@ def genHTMLLevel1(user_data, project_repos, hackathon_repos, blogs, stats_choice
     <img class="img-rounded" alt="Image" height="350px" src="{user_data['git_photo_url']}" />
     <h1 style='color:#f24b4b'> Hi, I am <a href="https://github.com/{user_data['username']}">{user_data['name']}</a>👨‍🎓 </h1>
     <h2 style='color:#21439e'>{user_data['git_bio']}</h2>
-    <!-- GITHUBSTATS-ENTRY -->  
+    <a href={resume_link} target="_blank"><h2 style='color:#d45131'>👉View Resume</h2></a>
```
Comment (kaustubhgupta) [inline themes/level_1/level1.py:19]: The href link needs to be in `""`.

Suggested change:
`"{resume_link}"`

File: themes/level_2/level2.py [hunk 3]
```diff
@@ -90,8 +90,9 @@ def genHTMLLevel2(user_data, project_repos, hackathon_repos, blogs, stats_choice
...
...3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"></path></svg></a>
```
Comment (kaustubhgupta) [inline themes/level_2/level2.py:93]: This is all wrong. The `{social_links}` needs no modification

File: themes/level_2/level2.py [hunk 4]
```diff
@@ -90,8 +90,9 @@ def genHTMLLevel2(user_data, project_repos, hackathon_repos, blogs, stats_choice
...
     </div>
+    <a href={resume_link} target="_blank"><h4 style='color:#d45131; margin:20px 0;'>View Resume</h4></a>
```
Comment (kaustubhgupta) [inline themes/level_2/level2.py:95]: Same change as mentioned in the first level

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: themes/level_1/level1.py (hunk 2 above)
Comment (ShivangiRai1310) [reply themes/level_1/level1.py]: Have added the quotes.

File: themes/level_2/level2.py (hunk 3 above)
Comment (ShivangiRai1310) [reply themes/level_2/level2.py]: Have removed the extra line.

File: generate_index.py (hunk 1 above)
Comment (ShivangiRai1310) [reply generate_index.py:92]: Tried to solve it considering the addBlogs.py file.
Have created addResume.py for the same.

Now it's working with False input, but upon giving the resume link as input it's still not showing the view resume part.

Please let me know where am I going wrong.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: themes/level_1/level1.py [hunk 5]
```diff
@@ -52,3 +54,6 @@ def genHTMLLevel1(user_data, project_repos, hackathon_repos, blogs, stats_choice
         return retunFile
     else:
         return template
+
+    if resume_link:
```
[the remark this answers is not in the document]
Comment (ShivangiRai1310) [reply themes/level_1/level1.py:58]: Okay got it. Thanks, I'll work on that.

**Comment by ShivangiRai1310** [comment]
Have done the changes. Please review.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@ShivangiRai1310 you did a good job of implementing this. That's why I have increased the points for this😃

**Pull request merged**